### PR TITLE
HL minor fixes

### PR DESCRIPTION
**Pull request by hilary-luo**
## Description

Minor changes to fix broken links, update install instructions and add some helpful notes.

## How Has This Been Tested?

Built and run as per the readme instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,31 @@ This repository contains the source code for building the Turtlebot4 User Manual
...
+```
+
+Next [install Jekyll and the rest of the prerequisites](https://jekyllrb.com/docs/). Be careful to not overwrite the Ruby version. 
 
 Clone this repository:
 
 ```bash
 git clone https://github.com/turtlebot/turtlebot4-user-manual.git
 ```
+Update the `_includes` folder locally to match [the Jekyll RTD Theme](https://github.com/rundocs/jekyll-rtd-theme/tree/develop/_includes) without overwriting any existing files.
```
Comment (roni-kreinin) [inline README.md]: Did this fix the issues for you? I had to clone the whole rtd-theme repo into the gem bundle to fix it.

Comment (hilary-luo) [reply README.md]: Interesting. Yah, this is all I had to do for it to make successfully. I repeated the process in a clean 22.04 VM too to make sure that I could follow this procedure step by step. That being said, I don't really have an opinion about which approach is better, they both work.

Comment (roni-kreinin) [reply README.md]: Can you add the required files from rtd-theme to this PR? That way we can just skip this step.

Comment (hilary-luo) [reply README.md]: Done and modified the instructions

File: software/overview.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ sort: 1
 
 # Overview
 
-The TurtleBot 4 runs on Ubuntu 20.04 LTS (Focal Fossa) and currently only supports [ROS 2 Galactic](https://docs.ros.org/en/galactic/index.html). The TurtleBot 4 software is entirely open source under the Apache 2.0 license, and is available on the [TurtleBot Github](https://github.com/turtlebot).
+The TurtleBot 4 runs on Ubuntu 20.04 LTS (Focal Fossa) or Ubuntu 22.04 (Jammy Jellyfish) and currently only supports [ROS 2 Galactic](https://docs.ros.org/en/galactic/index.html) or [ROS 2 Humble](https://docs.ros.org/en/humble/index.html) respectively. The TurtleBot 4 software is entirely open source under the Apache 2.0 license, and is available on the [TurtleBot Github](https://github.com/turtlebot).
 
 There are 2 main computers that run software used by TurtleBot 4: the onboard Raspberry Pi 4, the Create® 3 onboard processor. The user can also connect to the robot with their own PC to visualise sensor data, configure the robot, and more. Each computer is required to run Ubuntu 20.04 with ROS 2 Galactic.
```
Comment (roni-kreinin) [inline software/overview.md]: Replace "Each computer is required to run Ubuntu 20.04 with ROS 2 Galactic." with "Each computer is required to run the same version of Ubuntu and ROS 2 as the TurtleBot 4".

File: troubleshooting/faq.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ source /path/to/ws/install/setup.bash
 
 ### 2. Create® 3 topics are not visible
 
-First, check that the Create® 3 is connected to your WiFi network. You should be able to access the Create® 3 portal by entering the Create® 3 IP address in a browser. For information on how to connect the Create® 3 to WiFi, check the [quick start guide](../overview/quick_start.md#create%C2%AE-3-wifi-setup).
+First, check that the Create® 3 is connected to your WiFi network. You should be able to access the Create® 3 portal by entering the Create® 3 IP address in a browser. For information on how to connect the Create® 3 to WiFi, check the [quick start guide](../setup/simple_discovery.md#wi-fi-setup).
```
Comment (roni-kreinin) [inline troubleshooting/faq.md]: Change "quick start guide" to "Wi-Fi Setup guide"

File: tutorials/driving.md [hunk 4]
```diff
@@ -104,7 +108,7 @@ Set the `linear.x` value to drive the robot forwards or backwards, and the `angu
 
 ## Create® 3 Actions
 
-The Create® 3 provides a set of [ROS 2 Actions](https://docs.ros.org/en/galactic/Tutorials/Understanding-ROS 2-Actions.html) for driving the robot. You can use the [DriveDistance](https://github.com/iRobotEducation/irobot_create_msgs/blob/main/action/DriveDistance.action), [DriveArc](https://github.com/iRobotEducation/irobot_create_msgs/blob/main/action/DriveArc.action), and [RotateAngle](https://github.com/iRobotEducation/irobot_create_msgs/blob/main/action/RotateAngle.action) actions to tell the robot exactly how far and how fast to drive or rotate.
+The Create® 3 provides a set of [ROS 2 Actions](https://docs.ros.org/en/galactic/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Actions/Understanding-ROS2-Actions.html) for driving the robot. You can use the [DriveDistance](https://github.com/iRobotEducation/irobot_create_msgs/blob/main/action/DriveDistance.action), [DriveArc](https://github.com/iRobotEducation/irobot_create_msgs/blob/main/action/DriveArc.action), and [RotateAngle](https://github.com/iRobotEducation/irobot_create_msgs/blob/main/action/RotateAngle.action) actions to tell the robot exactly how far and how fast to drive or rotate.
```
Comment (roni-kreinin) [inline tutorials/driving.md]: Use the Humble link for ROS 2 Actions

**Review by roni-kreinin — CHANGES_REQUESTED** [review]

File: tutorials/driving.md (hunk 4 above)
Comment (hilary-luo) [reply tutorials/driving.md]: Done

File: troubleshooting/faq.md (hunk 3 above)
Comment (hilary-luo) [reply troubleshooting/faq.md]: Done

File: software/overview.md (hunk 2 above)
Comment (hilary-luo) [reply software/overview.md]: Done

**Pull request merged**